core_contracts/dividends/dividends.py

```diff
@@ -199,15 +199,6 @@ def on_install(self, _governance: Address) -> None:
         super().on_install()
         self._governance.set(_governance)
 
-        self._accepted_tokens.put(ZERO_SCORE_ADDRESS)
```

Comment:
Lets not remove these lines. This will help to track the changes we did while deploying the score.